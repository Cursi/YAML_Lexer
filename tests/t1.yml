---
# lists
simplelist: [e1,e2,1,2,3] #lala
empties: [[],[[],[]],[],[]]
non-empties: [[1,2],[[a,b],['5', '6', '7']],[single],[]]
