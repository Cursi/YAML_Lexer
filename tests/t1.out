| document: 
| | map value
| | | key: [simplelist]
| | | list value
| | | | string value: [e1]
| | | | string value: [e2]
| | | | numeric value: [1]
| | | | numeric value: [2]
| | | | numeric value: [3]
| | | key: [empties]
| | | list value
| | | | list value
| | | | list value
| | | | | list value
| | | | | list value
| | | | list value
| | | | list value
| | | key: [non-empties]
| | | list value
| | | | list value
| | | | | numeric value: [1]
| | | | | numeric value: [2]
| | | | list value
| | | | | list value
| | | | | | string value: [a]
| | | | | | string value: [b]
| | | | | list value
| | | | | | string value: [5]
| | | | | | string value: [6]
| | | | | | string value: [7]
| | | | list value
| | | | | string value: [single]
| | | | list value